src/game/Game.kt

```diff
@@ -28,9 +27,7 @@ class Game(private val player: Player, private val worldMap: WorldMap) {
         try {
             worldMap.moveObject(player.actor, 0, -1)
         } catch (ex: InvalidMoveException) {
```

Comment:
minor: this try-catch should probably span the entire 'when (playerchooseNextMove())' above, so that any exception related to processing of player's action gets fed back to the player.

Reply:
ok good idea